packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js

```diff
@@ -0,0 +1,150 @@
+    const { getState } = useFormApi();
+
+    const projectId = getState().values.authentication?.password;
+
+    return (
+        <TextContent>
+            <Text component={TextVariants.p}>
+                { intl.formatMessage({
+                    id: 'cost.gcp.dataset.description',
+                    defaultMessage: 'To collect and store the information needed for Cost Management, create a BigQuery dataset.'
+                }) }
+            </Text>
+            <TextList component={TextListVariants.ol}>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.dataset.firstStep',
+                        defaultMessage: 'From the GCP <b>BigQuery</b> console, select your project <b>({projectId})</b>.'
+                    }, { b, projectId })}
+                </TextListItem>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.dataset.secondStep',
+                        defaultMessage: 'Click <b>Create dataset.</b>'
+                    }, { b })}
+                </TextListItem>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.dataset.thirdStep',
+                        defaultMessage: 'Enter a name for your dataset in the <b>Dataset ID</b> field.'
+                    }, { b })}
+                </TextListItem>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.dataset.fourthStep',
+                        defaultMessage: 'Click <b>Create dataset.</b>'
+                    }, { b })}
+                </TextListItem>
+            </TextList>
+        </TextContent>
+    );
+};
+
+export const AssignAccess = () => {
+    const intl = useIntl();
+
+    return (
+        <TextContent>
+            <Text component={TextVariants.p}>
+                { intl.formatMessage({
+                    id: 'cost.gcp.assignAccessDesc',
+                    defaultMessage: 'To delegate account access, add a new member to your dataset.'
+                }) }
+            </Text>
+            <TextList component={TextListVariants.ol}>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.access.firstStep',
+                        defaultMessage: 'From your dataset, click on the button labeled <b>Share Dataset</b>.'
+                    }, { b })}
+                </TextListItem>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.access.secondStep',
+                        defaultMessage: 'Paste the following value into the <b>Add members</b> field:'
+                    }, { b })}
+                </TextListItem>
+                <ClipboardCopy className="pf-u-m-sm  pf-u-ml-0" isReadOnly>
+                    billing-export@red-hat-cost-management.iam.gserviceaccount.com
```

Comment:
Is it possible for the service account email to be seeded via an environment variable?

Reply:
Maybe, not sure. I have never done it. (Passing env variables to dependencies.)

Reply:
However, if you need to change this email according to env, we can do it here.

Reply:
We'll have a different service account email for production.  Do you have any suggestions on how best to handle this?

Reply:
If it isn’t going to be changed a lot we can hardcoded it here (if prod use this email). Or fetch it from an api. (As we are already fetching data for SubWatch from their api.)

Reply:
ok got it.  I think we might go the API fetching route.  We basically want to have one service account for development (CI/QA) and one for production and staging.

Reply:
@rvsia if we are going the hard-coded route initially can you update this to be:
`test-billing-service-account@cloud-billing-292519.iam.gserviceaccount.com`
Thanks!

Reply:
Ok, will change it next week!